CHANGELOG/CHANGELOG-0.2.md

```diff
@@ -0,0 +1,26 @@
+# Changelog 0.2
+
+## 0.2.0 - 2022-08-17
+
+### Added
```

Comment:
Some Grafana enhancement PRs may also be worth listing here. Reference: https://github.com/antrea-io/theia/issues/58

Reply:
+1. There are three related PRs, can be put in the `Changed` section.